arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1586,12 +1699,12 @@
       "x-ms-azure-resource": true,
       "description": "The Resource definition for other than namespace."
     },
-    "NamespaceListResult": {
+    "SBNamespaceListResult": {
```

Comment:
in PR: https://github.com/Azure/azure-rest-api-specs/pull/1036 and PR: https://github.com/Azure/azure-rest-api-specs/pull/959 its was suggested to rename NamespaceResource to Namespace which was conflicting as Namespace is reserved keyword. SB prefix is added to the conflicting entities like Namespace and Queue.